tests/integration/pages/experiment_overview.py

```diff
@@ -19,10 +43,121 @@ class ExperimentOverview(Base):
     )
     _overview_data_science_owner_locator = (
         By.CSS_SELECTOR,
-        "#id_analysis_owner"
+        "#id_analysis_owner",
     )
     _save_btn_locator = (By.CSS_SELECTOR, "#save-btn")
 
+    @property
+    def experiment_type(self):
+        options = self.find_elements(*self._experiment_type_locator)
+        for item in options:
+            if item.get_property("selected"):
+                text = item.text
+        if "addon" in text.replace("-", "").lower():
+            return "addon"
+        return "pref"
```

Comment:
Could you give me a psudo-code example?

Reply:
I could just have it return the name in a less hard-coded way

Reply:
Yeah exactly just return it like that and then in theory later instead of saying something like

    assert "pref" in exp_type

you'd do something like

    TYPE_PREF = "pref"

at the top of the file and then in your tests

    assert TYPE_PREF in exp_type

but this is fine as it is now 👍